pkg/apis/litmuschaos/v1alpha1/chaosengine_types.go

```diff
@@ -249,6 +249,8 @@ type RunProperty struct {
 	//ProbePollingInterval contains time interval, for which continuous probe should be sleep
 	// after each iteration
 	ProbePollingInterval int `json:"probePollingInterval,omitempty"`
+	//InitialDelaySeconds time interval for which probe will wait before run
+	InitialDelaySeconds int `json:"initialDelaySeconds,omitempty"`
```

Comment:
Is there no change in the auto-generated file?

Reply:
No, it changed only if we add/remove a struct

Reply:
okay